### ISSUE TITLE
Display Journal Data in Workflow Instance Viewer

### ISSUE THREAD

**Issue by sfmskywalker** [elsa-workflows/elsa-studio#66]
Just like in V2, the workflow instance viewer should display the Journal Data.

**Comment by ihtfw** [comment]
Also, it would be great to return the method addJournal so that it can log data in JavaScript.

**Comment by sfmskywalker** [comment]
Yes, makes sense. I'll open a separate issue for that, given that this issue is already here.

**Comment by sfmskywalker** [comment]
https://github.com/elsa-workflows/elsa-core/issues/6451
